Orange/widgets/visualize/owboxplot.py

```diff
@@ -1015,10 +1018,8 @@ def line(y0, y1):
         if self.compare == OWBoxPlot.CompareNone or len(self.stats) < 2:
             return
 
-        if self.compare == OWBoxPlot.CompareMedians:
-            crit_line = "median"
-        else:
-            crit_line = "mean"
+        crit_line = "median" if self.compare == OWBoxPlot.CompareMedians \
+            else "mean"
```

Comment:
This reduces the number of branches (radon wouldn't fall for this trick, but pylint does).

Reply:
You're right. I reverted this change. Even though it silenced the warning, the function was still too complicated and should be split into several functions, so the warning should stay.

Such, more complicated, issues should be fixed in separate PRs for each widget, though. This PR only deals with simple changes, otherwise it would be too big to review and merge.